components/KiboConfirmationDialog.vue

```diff
@@ -1,138 +1,81 @@
 <template>
-  <SfModal
-    class="sf-modal modal-confirm"
-    :visible="isOpen"
-    overlay
-    :persistent="false"
-    :cross="false"
-  >
-    <template #modal-bar>
-      <slot name="modal-bar" v-bind="{ label }">
-        <div class="title">
-          {{ label }}
-        </div>
-      </slot>
-    </template>
-    <transition name="sf-fade" mode="out-in">
-      <div class="modal-content">
-        <div class="action-buttons" v-bind="{ actionHandler }">
-          <button
-            class="action-buttons__cancel action-buttons__font"
-            :aria-disabled="false"
-            :link="null"
-            @click="$emit('click:close')"
-          >
-            {{ $t("Cancel") }}
-          </button>
-          <button
-            class="action-buttons__delete action-buttons__font"
-            :aria-disabled="false"
-            :link="null"
-            @click="actionHandler"
-          >
-            {{ $t("Delete") }}
-          </button>
-        </div>
+  <transition name="sf-fade" mode="out-in">
+    <div class="modal-content">
+      <div class="modal-content__header">{{ headerText }}</div>
+      <div class="action-buttons" v-bind="{ actionHandler }">
+        <SfButton class="color-light" :aria-disabled="false" :link="null" @click="$emit('onClose')">
+          {{ $t(buttonText1) }}
+        </SfButton>
+        <SfButton :aria-disabled="false" :link="null" @click="handleDelete">
+          {{ $t(buttonText2) }}
+        </SfButton>
       </div>
-    </transition>
-  </SfModal>
+    </div>
+  </transition>
 </template>
 <script lang="ts">
-import { SfModal } from "@storefront-ui/vue"
+import { SfButton } from "@storefront-ui/vue"
 export default {
   name: "KiboConfirmationDialog",
   components: {
-    SfModal,
+    SfButton,
   },
   props: {
-    isOpen: {
-      type: Boolean,
-      default: false,
+    buttonText1: {
```

Comment:
change buttonText1 to "cancelButtonText"
change buttonText2 to "confirmationButtonText"

Reply:
Done.